x/docs/md/help/executable.md

```diff
@@ -28,10 +28,10 @@ If the API key is incorrect, then re-run the `configure` command:
 exercism configure --key=YOUR_API_KEY
 ```
 
-Next look at the `Exercises Directory`. The current directory which is being used to fetch exercises can be viewed using the following command:
+Next look at the Exercises Directory. It will look something like this:
 
-```bash
-exercism configure --dir
+```plain
+Exercises Directory: /home/you/exercism
```

Comment:
There's no longer any context for this. In order to see it, you'd need to first call `exercism configure`.

That will show the full configuration, and then you can look for the line that says _Exercises Directory_.